cairo-contracts/packages/clients/src/cometbft/consensus_state.cairo

```diff
@@ -1,23 +1,56 @@
+use alexandria_data_structures::byte_array_ext::{ByteArrayIntoArrayU8, SpanU8IntoBytearray};
 use core::num::traits::Zero;
 use starknet_ibc_clients::cometbft::CometErrors;
 use starknet_ibc_core::client::{
     Duration, DurationTrait, Status, Timestamp, TimestampImpl, TimestampIntoU128, TimestampZero,
 };
 use starknet_ibc_core::commitment::{StateRoot, StateRootZero};
 
-#[derive(Clone, Debug, Drop, PartialEq, Serde, starknet::Store)]
+#[derive(Clone, Debug, Drop, PartialEq, Serde)]
 pub struct CometConsensusState {
     pub timestamp: Timestamp,
     pub root: StateRoot,
+    pub next_validators_hash: Array<u8>,
+}
+
+#[derive(Clone, Debug, Drop, PartialEq, starknet::Store)]
+pub struct CometConsensusStateStore {
+    pub timestamp: Timestamp,
+    pub root: StateRoot,
+    pub next_validators_hash: ByteArray,
+}
```

Comment:
I have two different structs here to be able to store `next_validator_hash` as `ByteArray` but use it as `Array<u8>`.